airbyte-webapp/src/components/connectorBuilder/Builder/AddStreamButton.module.scss

```diff
@@ -0,0 +1,23 @@
+@use "scss/variables";
+@use "scss/colors";
+
+.body {
+  display: flex;
+  flex-direction: column;
+  gap: variables.$spacing-xl;
+}
+
+.addButton {
```

Comment:
I really need the 9px padding to make the `+` look right in the add button, and we don't have a value in `_variables` for component sizes. But I did capture the 26px in a local variable for this file so that the px value isn't repeated